tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -280,7 +285,7 @@
             enable_debug: false
             # edpm firewall, change the allowed CIDR if needed
             edpm_sshd_configure_firewall: true
-            edpm_sshd_allowed_ranges: ['192.168.122.0/24']
+            edpm_sshd_allowed_ranges: "{{ ['192.168.122.0/24'] if dataplane_os_net_config_set_route|default(true)|bool else ['0.0.0.0/0'] }}"
```

Comment:
i think the quotes here are a mistake, and i will also remove the /0

`edpm_sshd_allowed_ranges: \"['0.0.0.0/0']\"\n        # SELinux module\n        edpm_selinux_mode: enforcing\n        plan: overcloud\nEOF\n", "delta": "0:00:00.761304", "end": "2023-11-30 08:13:50.213396", "msg": "", "rc": 0, "start": "2023-11-30 08:13:49.452092", "stderr": "+ oc apply -f -", "stderr_lines": ["+ oc apply -f -"], "stdout": "openstackdataplanenodeset.dataplane.openstack.org/openstack created", "stdout_lines": ["openstackdataplanenodeset.dataplane.openstack.org/openstack`